index.html

```diff
@@ -0,0 +1,219 @@
+                  </div>
+                  <div class="items">
+                    <div class="portofolio-img-4"></div>
+                    <h2 class="project-title">Tonic</h2>
+                      <div class="c-b-2015">
+                      <h6>CANOPY</h6>
+                      <ul class="c-b-2015-ul">
+                        <li>Back End Dev</li>
+                        <li>2015</li>
+                      </ul>
+                    </div>
+          
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="prog-lang">
+                      <li>html</li>
+                      <li>css</li>
+                      <li>javaScript</li>
+                    </ul>
+                    <a href="https://github.com/djo1975/HelloMicroverse" class="button">See Project</a>
+                  </div>
+            </div>
+            <section class="about-myself">
+
+                <section>
+                    <h1>About <br>Myself</h1>
+                    <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                <a>LET'S CONNECT</a>
+                <ul class="socialnetworks">
+                <li>
+                    <a href="https://twitter.com/home">
+                        <img src="twit.svg" alt="Twiter">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://www.linkedin.com/feed/">
+                        <img src="ln.svg" alt="Linkedin">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://medium.com/">
+                        <img src="m.svg" alt="Medium">
+                    </a>
+                </li>
+                <li>
+                    <a href="https://github.com/djo1975">
+                        <img src="cat.svg" alt="Github">
+                    </a>
+                </li>
+                <li>
+                   <a href="https://github.com/djo1975">
+                        <img src="v.svg" alt="Victory">
+                    </a>
+                </li>
+                </ul>
+                </div>
+                <div class="flex-item">
+                  <a href="" class="button resume-btn-border">Get my resume</a>
+                </div>
+        
+                <div class="flex-item">
+                  <ul class="rest">
+                      <li class="arrow"><img src="jsc.svg" alt="Down Arrow"></li>
+                    </ul>
+                    <section class="languages">
+                      <li><img src="js.svg"> JavaScript</li>
+                      <li><img src="html.svg"> HTML </li>
+                      <li><img src="css.svg">  CSS </li>
+                    </section>

```

Comment:
- For semantic reasons, when using the `li` tag, it is necessary that they are wrapped directly under a `ul` tag, please remove the `section` tag used in this part of your code and make sure your `li` tags are directly grouped under a `ul` tag. Please fix this 👍 👍